detection-rules/link_google_open_redirect_with_suspicious_indicators.yml

```diff
@@ -0,0 +1,81 @@
+name: "Open Redirect: Google domain with /url path and suspicious indicators"
+description: |
+  This rule examines messages containing image attachments that utilize Google's open redirect (Google.com/url..). 
+  To enhance accuracy and minimize false positives, the rule conducts additional assessments for suspicious indicators, as indicated in the comments.
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  // All attachments are images
+  and length(attachments) > 0
+  and all(attachments, .file_type in ('png', 'jpg', 'bmp'))
+  and sender.email.domain.root_domain not in $org_domains
```

Comment:
do we need to exclude legit google senders?

Reply:
It hasn't been an issue in the past, this [rule](https://github.com/sublime-security/sublime-rules/blob/8737eb18cce587e9442ed7d8b714af874ca4670d/detection-rules/link_google_amp_suspicious_indicators.yml) is very similar